README.md

```diff
@@ -1 +1,41 @@
-# js-capstone
\ No newline at end of file
+# mobile-portfolio
+This  project is aimed at building a beautiful porfolio page.
+
+# Built With
+HTML
+CSS
+JAVASCRIPT
+
+# Video presentation
+
+[MOVIES demo](https://youtu.be/PdEa4KJuPNk)
```

Comment:
- [ ] The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- [ ] The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
- [ ] Try to record a new video with more clear sounds